### DISCUSSION TITLE
Does latest version  drop defaultProps?

### DISCUSSION THREAD

**Discussion by kitkline** [react-native-elements/react-native-elements#3914] [Q&A]
Does anyone know if the latest version  of this library (noting it looks like no release in a year!) removes the use of defaultProps (which leads to the error 'Warning: TextElement: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead')

Is this package still under active development? If not what are people using instead (our app uses Input from this package extensively and we need to move off if this package is now not under active maintenance)

Cheers
Kit

**Comment by theianmay** [comment]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋 Thank you all for your patience while our new team has been hard at work. 
We’re excited to announce that **v5.0.0-beta.1** is now available! Release notes: https://github.com/react-native-elements/react-native-elements/releases/tag/v5.0.0-beta.1

This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.
